internal/storage/bucket_handle_test.go

```diff
@@ -138,3 +139,68 @@ func (t *BucketHandleTest) TestNewReaderMethodWithInvalidGeneration() {
 	AssertNe(nil, err)
 	AssertEq(nil, rc)
 }
+
+func (t *BucketHandleTest) TestDeleteObjectMethodWithValidObject() {
+	error := t.bucketHandle.DeleteObject(context.Background(),
+		&gcs.DeleteObjectRequest{
+			Name:                       TestObjectName,
+			Generation:                 0,
+			MetaGenerationPrecondition: nil,
+		})
+
+	AssertEq(nil, error)
+}
+
+func (t *BucketHandleTest) TestDeleteObjectMethodWithMissingObject() {
+	error := t.bucketHandle.DeleteObject(context.Background(),
+		&gcs.DeleteObjectRequest{
+			Name:                       missingObjectName,
+			Generation:                 0,
+			MetaGenerationPrecondition: nil,
+		})
+
+	err_expected := fmt.Errorf("storage: object doesn't exist")
+	AssertEq(err_expected.Error(), error.Error())
+}
+func (t *BucketHandleTest) TestDeleteObjectMethodWithValidGeneration() {
```

Comment:
Please add a separate line in b/w the methods, this improves the readability.

Reply:
fixed